src/domUpdates.js

```diff
@@ -190,6 +189,11 @@ const updateActiveRecipes = (currentUser, data) => {
 //   //but
 // });
 
+const displayErrorMessage = (userErrorMessage) => {
+  errorMessage.classList.toggle("hidden", false);
```

Comment:
Brilliant. I was thinking of making another display error function and using that inner text.